content/articles/generating-data-for-a-random-walk-using-python/index.md

```diff
@@ -0,0 +1,145 @@
+In this article, we'll use Python to collect data for a random walk and then use matplotlib to visualize the results. A random walk is a route with no predefined way that is determined by a sequence of random choices made totally at random. A random walk could be compared to the path a cockroach may follow if it lost its mind and takes each walk differently. 
+
+### Prerequisites
+ - Have some basic knowledge of [Python programming language](https://docs.microsoft.com/en-us/windows/python/beginners)
+ - Have [Python](https://www.journaldev.com/30076/install-python-windows-10) and [matplotlib](https://www.tutorialspoint.com/how-to-install-matplotlib-in-python) installed in your machine.
+ - Have some basic understanding of [matplotlib](https://www.simplilearn.com/tutorials/python-tutorial/matplotlib)
+
+### Creating the RWalk() Class
+To design a random walk, we'll construct an `RWalk` class that generates random choices on the basis the trek should go in. Three characteristics are required for the class: a single variable to record the amount of instances in the path, and two arrays to hold the x- and y- data of each step in the trek. The `RWalk` class will only have two functions:` __init__()` and `fil_walk()`, which will determine the steps in the trek. Let's construct a file named `rwalk.py` and begin with the ` __init__()` function as illustrated below:
```

Comment:
`Three characteristics are required for the class:`

Make the 3 characteristics as a numbered list.

Reply:
I find only 2 characteristics. Did you miss any?